src/main/bash/sdkman-main.sh

```diff
@@ -155,8 +152,20 @@ function sdk() {
 			"$native_command" "$QUALIFIER"
 		elif [ -z "$4" ]; then
 			"$native_command" "$QUALIFIER" "$3"
-		else
+		elif [ -z "$5" ]; then
 			"$native_command" "$QUALIFIER" "$3" "$4"
+		elif [ -z "$6" ]; then
+			"$native_command" "$QUALIFIER" "$3" "$4" "$5"
```

Comment:
Could you just use a slice, like `"${@:3}"`? Should also work for the other cases.